activerecord/lib/active_record/associations/collection_proxy.rb

```diff
@@ -1033,8 +1005,9 @@ def records # :nodoc:
       end
 
       # Adds one or more +records+ to the collection by setting their foreign keys
-      # to the association's primary key. Returns +self+, so several appends may be
-      # chained together.
+      # to the association's primary key. Since #<< flattens its argument list and
```

Comment:
```suggestion
      # to the association's primary key. Since <tt><<</tt> flattens its argument list and
      # inserts each record, +push+ and +concat+ behave identically. Returns +self+
```